mbedtls/src/ssl/context.rs

```diff
@@ -126,6 +126,8 @@ pub struct Context<T> {
     client_transport_id: Option<Vec<u8>>,
 }
 
+unsafe impl<T: Sync> Sync for Context<T> {}
```

Comment:
@Taowyoo let me know if you're comfortable reviewing this change, or we can find someone else